wolfssl/wolfcrypt/wc_port.h

```diff
@@ -1375,6 +1375,15 @@ WOLFSSL_ABI WOLFSSL_API int wolfCrypt_Cleanup(void);
         #endif
     #endif
 
+#if defined(__STRICT_ANSI__)
+    #define WOLFSSL_FLEXIBLE_ARRAY_SIZE 1
+#elif defined(__GNUC__)
+    #define WOLFSSL_FLEXIBLE_ARRAY_SIZE
+    /*!< \brief Value appropriate as a size for an array that will be allocated to a variable size.  Built-in value usually works. */
+#else
+    #define WOLFSSL_FLEXIBLE_ARRAY_SIZE 0
+#endif
+
```

Comment:
The latest setup from wolfSentry has more subtleties.

See https://github.com/wolfSSL/wolfsentry/blob/46af28fcfc7556a4db08b7a0f14068e5ab645413/wolfsentry/wolfsentry_settings.h#L327